ethereum/test/test_eth_app.js

```diff
@@ -69,14 +70,57 @@ describe("ETHApp", function () {
 
       event.args.sender.should.be.equal(userOne);
       event.args.recipient.should.be.equal(POLKADOT_ADDRESS);
+      BigNumber(event.args.paraId).should.be.bignumber.equal(0);
+      BigNumber(event.args.fee).should.be.bignumber.equal(0);
       BigNumber(event.args.amount).should.be.bignumber.equal(amount);
 
       // Confirm contract's balance has increased
       const afterBalance = await web3.eth.getBalance(this.app.address);
       afterBalance.should.be.bignumber.equal(beforeBalance.plus(amount));
 
     });
-  })
+
+    it("should lock funds to destination parachain", async function () {
+      const beforeBalance = BigNumber(await web3.eth.getBalance(this.app.address));
+      const amount = BigNumber(web3.utils.toWei("0.25", "ether"));
+
+      const tx = await lockupFunds(this.app, userOne, POLKADOT_ADDRESS, amount, ChannelId.Basic, 1001, 4_000_000)
+        .should.be.fulfilled;
+
+      // Confirm app event emitted with expected values
+      const event = tx.logs.find(
+        e => e.event === "Locked"
+      );
+
+      event.args.sender.should.be.equal(userOne);
+      event.args.recipient.should.be.equal(POLKADOT_ADDRESS);
+      BigNumber(event.args.paraId).should.be.bignumber.equal(1001);
+      BigNumber(event.args.fee).should.be.bignumber.equal(4_000_000);
+      BigNumber(event.args.amount).should.be.bignumber.equal(amount);
+
+      // Confirm contract's balance has increased
+      const afterBalance = await web3.eth.getBalance(this.app.address);
+      afterBalance.should.be.bignumber.equal(amount);
+
+      // Confirm contract's locked balance state has increased by amount locked
+      const afterBalanceState = BigNumber(await web3.eth.getBalance(this.app.address));
+      afterBalanceState.should.be.bignumber.equal(beforeBalance.plus(amount));
+    });
+
+    it("should not lock funds to destination parachain without fee", async function () {
```

Comment:
ditto, as per my comment above, the contract shouldn't really do any verification of the fee.

Reply:
Removed tests.